osgi/src/main/resources/resources/ehcache-cda.xml

```diff
@@ -0,0 +1,170 @@
+                                         element is never expired.
+        overflowToDisk                 - Sets whether elements can overflow to disk when the in-memory cache
+                                         has reached the maxInMemory limit.
+
+        The following attributes are optional:
+
+		eternal                        - Ignore timeouts, elements never expire.
+        timeToIdleSeconds              - Sets the time to idle for an element before it expires.
+                                         i.e. The maximum amount of time between accesses before an element expires
+                                         Is only used if the element is not eternal.
+                                         Optional attribute. A value of 0 means that an Element can idle for infinity.
+                                         The default value is 0.
+        timeToLiveSeconds              - Sets the time to live for an element before it expires.
+                                         i.e. The maximum time between creation time and when an element expires.
+                                         Is only used if the element is not eternal.
+                                         Optional attribute. A value of 0 means that and Element can live for infinity.
+                                         The default value is 0. (Overriden in .cda files)
+        diskPersistent                 - Whether the disk store persists between restarts of the Virtual Machine.
+                                         The default value is false.
+        diskExpiryThreadIntervalSeconds- The number of seconds between runs of the disk expiry thread. The default value
+                                         is 120 seconds.
+
+        memoryStoreEvictionPolicy      - LRU: Least Recently Used (Default); 
+                                         LFU: Less Frequently Used; 
+                                         FIFO: First In First Out.
+        maxElementsOnDisk              - Sets the maximum number of objects that will be kept on disk. Evictions from
+                                         disk will always use the LFU algorithm.
+        diskSpoolBufferSizeMB          - This is the size to allocate the DiskStore for a spool buffer. Writes are made
+                                         to this area and then asynchronously written to disk. The default size is 30MB.
+																					
+        -->	
+    <diskStore path="java.io.tmpdir/_pentaho-cda"/>
+    
+	<!--terracottaConfig url="localhost:9510"/-->
+
+	<defaultCache
+        maxElementsInMemory="1000"
+        eternal="false"
+        overflowToDisk="false"
+        timeToIdleSeconds="0"
+        timeToLiveSeconds="0"
+        diskPersistent="false"
+        diskExpiryThreadIntervalSeconds="120"
+     />
+        
+	<cache name="pentaho-cda-dataaccess"
+        maxElementsInMemory="50"
```

Comment:
Any reasoning for the changes here? I noticed the cda-pentaho version has `maxElementsInMemory="100" maxElementsOnDisk="10000"`.